### ISSUE TITLE
Update accessibility statement after Frontend V5 release

### ISSUE THREAD

**Issue by dav-idc** [alphagov/govuk-design-system#3129]
## What
Remove items from the accessibility statement once V5 goes out. Or possibly, add a section on recently fixed pieces?

These issues will be resolved by V5:
- https://github.com/alphagov/govuk-design-system/issues/2680
- https://github.com/alphagov/govuk-frontend/issues/3673
- https://github.com/alphagov/govuk-frontend/issues/3674

These issues will not be resolved, and should be left on the statement:
- https://github.com/alphagov/govuk-frontend/issues/2472
- https://github.com/alphagov/govuk-frontend/issues/3693
- https://github.com/alphagov/govuk-frontend/issues/3686

Resources:
- [Accessibility statement page](https://design-system.service.gov.uk/accessibility/)
- [Previous accessibility statement update Google Doc](https://docs.google.com/document/d/1Isr0WwmCZayCs9sTgyizzsJVKF0rRL_UUVo8lQrtn0E/edit)
- [Previous PR for updating the accessibility statement](https://github.com/alphagov/govuk-design-system/pull/2995)
## Why
Once V5 is published, some of the issues we list in our accessibility statement should be resolved. For cleanliness and transparency, we should update the statement and remove those items.
## Who needs to work on this
David (accessibility), content designer
## Who needs to review this
Product manager? Delivery manager?
## Done when

- [x] Statement update is drafted
- [x] Statement update is reviewed by team
- [x] Statement update is published
- [x] Related GitHub issues are closed

**Comment by stevenjmesser** [comment]
Steve to check with Frontend squad when they think v5 will be ready.

**Comment by claireashworth** [comment]
Latest [accessibility statement draft](https://docs.google.com/document/d/1RYGVRZQOHkr_a_nsjmy8oWS3fvOZhQBr-d6W-C5ioMA/edit?usp=sharing) underway

**Comment by claireashworth** [comment]
Amendments checked with Charlotte and draft shared with team.

**Comment by romaricpascal** [comment]
@claireashworth Lost track of what's merged and not, did this one go in the big v5 content merge or will it go on its own now we've released. Feel free to close and set to Done if the former 😊

**Comment by stevenjmesser** [comment]
I've suggested one change as we were saying that that the Design System website was partially compliant but that nothing on the website was non-compliant, which doesn't make sense.

Have also suggested some (bad) wording for including mention of compliance against WCAG 2.2, but Claire helpfully pointed out we might have to stick to the [template language](https://www.gov.uk/guidance/model-accessibility-statement). 

Have finished my review and shall pass this back to Claire for writing the WCAG 2.2 lines, and move it back to In Progress.

**Comment by claireashworth** [comment]
The statement is redrafted in line with the latest accessibility template and ready for another review.

**Comment by stevenjmesser** [comment]
Reviewed. Moving back to in progress so that [accessibility.md](https://github.com/alphagov/govuk-design-system/blob/main/src/accessibility.md) can be updated and a PR raised.

**Comment by claireashworth** [comment]
Accessibility statement amended in PR '[WCAG 2.2 guidance updates #3090](https://github.com/alphagov/govuk-design-system/pull/3090/commits/573f1367d4222311556628672ae9207ffe24aab6)'
@calvin-lau-sig7 @CharlotteDowns for info/to review
This is the active issue for the accessibility statement. I've also commented on #3332, but this might be a duplicate issue.

**Comment by stevenjmesser** [comment]
It's published.
